With Go 1.16, "make" cannot install external plugins

**What happened**:

With [Go 1.16](https://golang.org/doc/go1.16#go-command):

> Build commands like go build and go test no longer modify go.mod and go.sum by default. Instead, they report an error if a module requirement or checksum needs to be added or updated (as if the -mod=readonly flag were used). Module requirements and sums may be adjusted with go mod tidy or go get. 

For that reason, using `make` to install external plugins does not work.

Example:

```
$ echo "mysql:github.com/cloud66-oss/coredns_mysql" >> plugin.cfg
$ make
CGO_ENABLED=0  go build -v -ldflags="-s -w -X github.com/coredns/coredns/coremain.GitCommit=a644eb44-dirty" -o coredns
core/plugin/zplugin.go:7:2: no required module provides package github.com/cloud66-oss/coredns_mysql; to add it:
	go get github.com/cloud66-oss/coredns_mysql
make: *** [Makefile:16: coredns] Error 1
```

**What you expected to happen**:

`make` should compile coredns successfully, adding the external plugin module name (here `github.com/cloud66-oss/coredns_mysql`) to `go.mod`.

**How to reproduce it (as minimally and precisely as possible)**:

1. Clone CoreDNS
1. Add an external plugin. E.g.
    ```
    echo "mysql:github.com/cloud66-oss/coredns_mysql" >> plugin.cfg
    ```
1. Compile:
    ```
    make
    ```

**Anything else we need to know?**:

**Environment**:

- the version of CoreDNS: master branch
- Corefile: N/A
- logs, if applicable: N/A
- OS (e.g: `cat /etc/os-release`): Fedora 34
- Others:
  - go version go1.16.6 linux/amd64
  - `GO111MODULE=""`

--- a/Makefile
+++ b/Makefile
@@ -20,10 +20,12 @@ check: core/plugin/zplugin.go core/dnsserver/zdirectives.go
 
 core/plugin/zplugin.go core/dnsserver/zdirectives.go: plugin.cfg
 	go generate coredns.go
+	go get
 
 .PHONY: gen
 gen:
 	go generate coredns.go
+	go get
 
 .PHONY: pb
 pb:

